prov/efa/test/efa_unit_test_info.c

```diff
@@ -116,11 +116,6 @@ static void test_info_direct_attributes_impl(struct fi_info *hints,
 		assert_false(info->tx_attr->msg_order & FI_ORDER_SAS);
 		assert_int_equal(info->domain_attr->progress, FI_PROGRESS_AUTO);
 		assert_int_equal(info->domain_attr->control_progress, FI_PROGRESS_AUTO);
-		assert_int_equal(
-			g_efa_selected_device_list[0].rdm_info->ep_attr->max_msg_size,
-			(info->caps & FI_RMA) ?
-				g_efa_selected_device_list[0].max_rdma_size :
-				g_efa_selected_device_list[0].ibv_port_attr.max_msg_sz);
```

Comment:
the commit message doesn't sound clear to me, can you elaborate in what scenario this check will fail?